### PR TITLE
Docs simple project

### PR DESCRIPTION
**Pull request by eysi09**

### CODE REVIEW

**Comment by eysi09** [comment]
This needs to be updated pending some of the PRs we have open.

File: examples/simple-project-start/services/go-service/webserver/main.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ func handler(w http.ResponseWriter, r *http.Request) {
 }
```
Comment (ellenkorbes) [inline examples/simple-project-start/services/go-service/webserver/main.go]: Here shouldn't we just stick to simply `fmt.Fprintf(w, "Hello from Go!")`? Also there's an extra blank space on line 226 on `simple-project.md`.

Comment (eysi09) [reply examples/simple-project-start/services/go-service/webserver/main.go]: Good catch. I've updated accordingly.

File: examples/local-tls/services/go-service/webserver/main.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Hello %s from Go!", r.URL.Path[1:])
+	fmt.Print("Hello from Go!")
 }
```
Comment (ellenkorbes) [inline examples/local-tls/services/go-service/webserver/main.go]: `fmt.Print` will print to the screen, I suggest `fmt.Fprint` into `w` so it prints into the http response.

Comment (eysi09) [reply examples/local-tls/services/go-service/webserver/main.go]: Ah yes. Fixed.

**Pull request merged**